os/kernel/Makefile

```diff
@@ -68,6 +68,8 @@ include errno/Make.defs
 include wdog/Make.defs
 include semaphore/Make.defs
 include binary_manager/Make.defs
+include binary_manager/compression/Make.defs
+include binary_manager/compression/lzma/Make.defs
```

Comment:
Do we need to check  CONFIG_COMPRESSED_BINARY here?

Reply:
ok..  will add config check here

Reply:
done